src/agri_ai/utils/error_handling.py

```
@@ -0,0 +1,260 @@
+"""
+エラーハンドリングユーティリティ
+"""
+
+import logging
+import functools
+from typing import Callable, Any, Optional, Type, Union
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)